package.json

```diff
@@ -37,6 +37,7 @@
     "babel-preset-stage-0": "^6.0.15",
     "bootstrap-loader": "^2.0.0-beta.21",
     "bootstrap-sass": "^3.3.7",
+    "butter-base-theme": "^0.1.2",
```

Comment:
our naming scheme is `butter-$thing-$name`; `butter-component-settings`, `butter-streamer-http`, `butter-theme-blah`